paddle/fluid/operators/optimizers/lamb_op.h

```diff
@@ -125,13 +146,24 @@ struct SparseLambMomentUpdateFunctor {
     T mom1 = moment1_[i];
     T mom2 = moment2_[i];
     T p = param_[i];
+    T beta1_pow = *beta1_pow_;
```

Comment:
Why do you need to initialize `T beta1_pow` and `T beta2_pow` here? Can you use pointer directly?

Reply:
Thanks, done.